change/@fluentui-react-ef007003-8eb0-4f9c-9f4d-71de8aa28b6e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

This can be type `patch` since it's not adding any public-facing APIs. (The issue label was a little misleading in this case, sorry.)
```suggestion
  "type": "patch",
```